featurebyte/models/feature.py

```diff
@@ -78,6 +78,7 @@ class BaseFeatureModel(FeatureByteCatalogBaseDocumentModel):
 
     # list of IDs attached to this feature or target
     entity_ids: List[PydanticObjectId] = Field(allow_mutation=False, default_factory=list)
+    primary_entity_ids: List[PydanticObjectId] = Field(allow_mutation=False, default_factory=list)
```

Comment:
Does this have to be marked as Optional for backward compatibility?

Reply:
The `default_factory=list` is set here. If this field does not exist, it will be generated as empty list.